src/dom/attributes/attribute-manager.js

```diff
@@ -9,6 +9,55 @@
  *
  * The Attribute Manager is specific to adding custom attributes and describing their implementation (handlers). (The [Dom Manager](../../) contains the general implementation.)
  *
+ *
+ * **Examples**
+ *
+ * Built-in attribute managers like `data-f-value` and `data-f-foreach` automatically bind variables in your project's model to particular HTML elements. However, the value of the variable itself isn't always what you want in the UI.
```

Comment:
Maybe reword with something like "However, your use-case may sometimes require "doing something" with the value of the variable, instead of just printing it out. " (also feel free to keep as-is)